Source/Synthesis.Tests/Fixtures/ContentSearch/StandardTemplateTests.cs

```diff
@@ -200,11 +204,18 @@ public void ContentSearch_FindsItem_ByCustomSearchFieldIndexer()
 		{
 			using (var context = CreateTestSearchContext())
 			{
-				var query = context.GetSynthesisQueryable<IStandardTemplateItem>()
+				var query = ResolveSynthesisQueryable(context)
 					.Where(x => x.TemplateId == TemplateIDs.Sublayout && x["placeholder"] == "content");
 
 				Assert.GreaterOrEqual(query.Count(), 1, "Could not find a sublayout item with placeholder set to 'content'!");
 			}
 		}
+
```

Comment:
This should be unnecessary once we add a DI model to GetSynthesisQueryable (see https://github.com/kamsar/Synthesis/pull/21/files#r32533998)